net/instaweb/rewriter/resource_fetch_test.cc

```diff
@@ -43,6 +43,7 @@ namespace {
 
 const char kCssContent[] = "* { display: none; }";
 const char kMinimizedCssContent[] = "*{display:none}";
+const GoogleString kValue("Value");
```

Comment:
const char kValue[] = "Value";

We don't statically construct class instances, and in this case it's not needed anyway.